assets/js/src/storybook/_stories/header.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+export default {
+  title: 'Typography',
+};
+
+export const Headers = () => (
+  <>
+    <h1>&lt;h1&gt; header</h1>
```

Comment:
I thought the idea was to create react components for the elements so that we would use: `<Heading level={1}>Header</Heading>` or something like that. Again Gutenberg does that https://developer.wordpress.org/block-editor/components/text/